Sources/DeepLearning/Loss.swift

```diff
@@ -74,13 +74,17 @@ func _vjpSoftmaxCrossEntropy<Scalar: TensorFlowFloatingPoint>(
 /// Computes the sigmoid cross entropy (binary cross entropy) between logits and labels.
 ///
 /// - Parameters:
-///   - logits: Single continuous values from `0` to `1`.
+///   - logits: Unscaled value from a neural network.
 ///   - labels: Integer values that correspond to the correct output.
-@differentiable
+@differentiable(wrt: logits)
 public func sigmoidCrossEntropy<Scalar: TensorFlowFloatingPoint>(
     logits: Tensor<Scalar>, labels: Tensor<Scalar>
```

Comment:
Should there be 2 generic types, so that `labels` can be a Tensor of a non-`TensorFlowFloatingPoint` type?

Reply:
I don't think so. Labels need to be multiplied with logits (see line 88).

Reply:
I don't think so. Labels need to be multiplied with logits (see line 88).